RELEASE/scripts/autoscend/combat/auto_combat_util.ash

```diff
@@ -57,16 +57,32 @@ boolean canUse(skill sk, boolean onlyOnce)
 		return false;
 	}
 
-	if(my_mp() < mp_cost(sk) - combat_mana_cost_modifier() ||
+	if(inCombat)
+	{
+		if(my_mp() < mp_cost(sk) - combat_mana_cost_modifier() ||
```

Comment:
combat_mana_cost_modifier() seems to return a negative value when skills cost less mana. Believe this should be mp_cost(sk) + combat_mana_cost_modifier() 

Small test:
1. combat_mana_cost_modifier() returned 0
2. used natural magick candle
3. combat_mana_cost_modifier() returned -3

Reply:
Amusingly this is old code I did not write.
Nice catch, I am wondering if this returning of a negative is related to issue #71

Reply:
done